mapproxy/config/loader.py

```diff
@@ -1951,13 +1951,20 @@ def tile_layers(self, grid_name_as_path=False):
                 md['format'] = self.context.caches[cache_name].image_opts().format
                 md['cache_name'] = cache_name
                 md['extent'] = extent
+                legend_version = None
+                if 'legendurl' in self.conf:
+                    wms_conf = self.context.services.conf.get('wms', None)
+                    # GetLegendGraphic with legendurl does not seem to work in non 1.3.0 versions
```

Comment:
Maybe add this to the documentation of the legendUrl parameter.

Reply:
This is a bug in MapProxy and should be changed as soon as the bug is addressed. Alternatively we could add code here that chooses the highest version number so it is assuming a correctly working `GetLegendGraphic` and works as soon as the bug is fixed.

I opened this issue: https://github.com/mapproxy/mapproxy/issues/946

Reply:
I changed it so it should be fixed as soon as the issue is addressed